dbOperations/removeUser.js

```diff
@@ -13,48 +14,46 @@ import { getPfpName } from "~/Utility";
 
 async function removeUser(teamId, userId) {
   try {
-    await runTransaction(db, async (transaction) => {
-      //Remove all attempts from attempts table with UID == userID
-      const attemptQuery = query(
-        collection(db, "teams", teamId, "attempts"),
-        where("uid", "==", userId),
-      );
+    //Remove all attempts from attempts table with UID == userID
+    const attemptQuery = query(
+      collection(db, "teams", teamId, "attempts"),
+      where("uid", "==", userId),
+    );
 
-      const attemptSnapshot = await getDocs(attemptQuery);
+    const attemptSnapshot = await getDocs(attemptQuery);
 
-      for (const doc of attemptSnapshot.docs) {
-        await transaction.delete(doc.ref);
-      }
+    for (const doc of attemptSnapshot.docs) {
+      await deleteDoc(doc.ref);
+    }
 
-      //Remove all entries from best_attempts table with UID == userID
-      const bestAttemptQuery = query(
-        collection(db, "teams", teamId, "best_attempts"),
-      );
+    //Remove all entries from best_attempts table with UID == userID
+    const bestAttemptQuery = query(
+      collection(db, "teams", teamId, "best_attempts"),
+    );
 
-      const bestAttemptSnapshot = await getDocs(bestAttemptQuery);
+    const bestAttemptSnapshot = await getDocs(bestAttemptQuery);
 
-      for (const doc of bestAttemptSnapshot.docs) {
-        const docData = doc.data();
+    for (const doc of bestAttemptSnapshot.docs) {
+      const docData = doc.data();
 
-        if (docData[userId]) {
-          //Delete the field
-          await transaction.update(doc.ref, {
-            [userId]: deleteField(),
-          });
-        }
+      if (docData[userId]) {
+        //Delete the field
+        await updateDoc(doc.ref, {
+          [userId]: deleteField(),
+        });
       }
+    }
 
-      //Remove user from user table where UID == userID
-      const userRef = doc(db, "teams", teamId, "users", userId);
+    //Remove user from user table where UID == userID
+    const userRef = doc(db, "teams", teamId, "users", userId);
 
-      await transaction.delete(userRef);
+    await deleteDoc(userRef);
 
-      await removePfp(getPfpName(teamId, userId));
+    await removePfp(getPfpName(teamId, userId));
```

Comment:
I think this is throwing an error, the user gets deleted but an error message keeps popping up:
![image](https://github.com/user-attachments/assets/ccf0f456-8db2-43c0-a131-aeac0b527a19)